command_line_intro/files.md

```diff
@@ -0,0 +1,23 @@
+# Files and Directories
```

Comment:
This is a good overview of shells, but the title made me expect more stuff like ls/cd/mkdir/rm/cat/more/etc.

Reply:
that's definitely a markdown error.... https://github.com/pulibrary/ruby_for_metadataists/blob/command_line_intro/command_line_intro/sec2/file_and_directory.md